### DISCUSSION TITLE
在浏览器播放效果与PAGViewer、AE的效果不一样，浏览器中描边会消失

### DISCUSSION THREAD

**Discussion by yinjiaoyuan** [Tencent/libpag#2517] [General]
在浏览器播放效果与PAGViewer、AE的效果不一样，浏览器中描边会消失。
在AE中的效果是：
![image](https://github.com/user-attachments/assets/3b3d3d11-a65b-47a1-95ab-084c3c91df66)
在PAGViewer的效果是：
![img_v3_02fg_dce1a0f5-5e62-4b95-8fc7-f0dcf759d4dg](https://github.com/user-attachments/assets/9174cc84-f8f5-46c9-baae-0c8697a6ef67)
在线播放的效果是：
![img_v3_02fg_a5f20e0b-a201-4e43-a8ee-eb3f957a797g](https://github.com/user-attachments/assets/1bea080c-a4c2-45f1-8c6b-90fcd57e526f)
pag文件是：
https://github.com/yinjiaoyuan/demo/blob/main/hecheng2.pag

**Comment by CodeJhF** [comment]
同类问题已由 #2890 继续跟踪，为避免重复排查，先关闭本条。
如果有新的复现信息，欢迎到该帖补充，或在本帖回复后重开。（stale-close）
